<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>绘图软件</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MyGraphicsView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="upToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="bigAction"/>
   <addaction name="smallAction"/>
   <addaction name="restoreAction"/>
   <addaction name="separator"/>
   <addaction name="leftAction"/>
   <addaction name="rightAction"/>
   <addaction name="forwardAction"/>
   <addaction name="backwardAction"/>
   <addaction name="combineAction"/>
   <addaction name="divorceAction"/>
   <addaction name="separator"/>
   <addaction name="deleteAction"/>
   <addaction name="separator"/>
   <addaction name="quitAction"/>
  </widget>
  <widget class="QToolBar" name="leftToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="rectangleAction"/>
   <addaction name="ellipseAction"/>
   <addaction name="circleAction"/>
   <addaction name="triangleAction"/>
   <addaction name="trapezoidAction"/>
   <addaction name="lineAction"/>
   <addaction name="textAction"/>
  </widget>
  <action name="bigAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图元</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="smallAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图元</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="restoreAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/restore.png</normaloff>:/images/restore.png</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="leftAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="text">
    <string>左旋转</string>
   </property>
   <property name="toolTip">
    <string>左旋转</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="rightAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="text">
    <string>右旋转</string>
   </property>
   <property name="toolTip">
    <string>右旋转</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="forwardAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/foreground.png</normaloff>:/images/foreground.png</iconset>
   </property>
   <property name="text">
    <string>前置</string>
   </property>
   <property name="toolTip">
    <string>居于最前面</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="backwardAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/background.png</normaloff>:/images/background.png</iconset>
   </property>
   <property name="text">
    <string>后置</string>
   </property>
   <property name="toolTip">
    <string>居于最后面</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="combineAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/combine.png</normaloff>:/images/combine.png</iconset>
   </property>
   <property name="text">
    <string>组合</string>
   </property>
   <property name="toolTip">
    <string>组合</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="divorceAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/divorce.png</normaloff>:/images/divorce.png</iconset>
   </property>
   <property name="text">
    <string>拆开</string>
   </property>
   <property name="toolTip">
    <string>拆开</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除图元</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/leave.png</normaloff>:/images/leave.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="rectangleAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/rectangle.png</normaloff>:/images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="toolTip">
    <string>矩形</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="ellipseAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/ellipse.png</normaloff>:/images/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
   <property name="toolTip">
    <string>椭圆</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="circleAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/ball.png</normaloff>:/images/ball.png</iconset>
   </property>
   <property name="text">
    <string>圆形</string>
   </property>
   <property name="toolTip">
    <string>圆</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="triangleAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/triangle.png</normaloff>:/images/triangle.png</iconset>
   </property>
   <property name="text">
    <string>三角形</string>
   </property>
   <property name="toolTip">
    <string>三角形</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="trapezoidAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/poly.png</normaloff>:/images/poly.png</iconset>
   </property>
   <property name="text">
    <string>梯形</string>
   </property>
   <property name="toolTip">
    <string>梯形</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="lineAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="toolTip">
    <string>直线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="textAction">
   <property name="icon">
    <iconset resource="../resources/res.qrc">
     <normaloff>:/images/text.png</normaloff>:/images/text.png</iconset>
   </property>
   <property name="text">
    <string>文字</string>
   </property>
   <property name="toolTip">
    <string>文字</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>MyGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
